README.md

```diff
@@ -35,6 +35,51 @@ submit a pull request.
 Please see the [CONTRIBUTING](CONTRIBUTING.md) file in this repository for
 more details
 
+## Supported Capabilities
+
+| Capability     |   Windows  |  Linux  |  Mac  |   Android   |
+| -------------- | :--------: | :-----: | :-------: | :-----: |
+| **Vulkan Support** |||||
+| Capture | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: |
+| Replay  | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: |
+| gfxrecon-compress | :white_check_mark: | :white_check_mark: | :white_check_mark: | :large_orange_diamond: |
+| gfxrecon-convert | :white_check_mark: | :white_check_mark: | :white_check_mark: | :large_orange_diamond: |
+| gfxrecon-info | :white_check_mark: | :white_check_mark: | :white_check_mark: | :large_orange_diamond: |
+| gfxrecon-optimize | :white_check_mark: | :white_check_mark: | :white_check_mark: | :large_orange_diamond: |
+| gfxrecon-tocpp | :x: | :construction: | :x: | :construction:  |
+| **D3D12 Support** |||||
+| Capture  | :white_check_mark: | :x: | :x: | :x: |
+| Replay   | :white_check_mark: | :x: | :x: | :x: |
+| gfxrecon-compress | :white_check_mark: | :x: | :x: | :x: |
+| gfxrecon-convert | :white_check_mark: | :x: | :x: | :x: |
+| gfxrecon-info | :white_check_mark: | :x: | :x: | :x: |
+| gfxrecon-optimize | :white_check_mark: | :x: | :x: | :x: |
+
+**Legend**
+* :white_check_mark: : Supported
+* :x: : Not Suported
+* :large_orange_diamond: : Not supported on Android, but Desktop tools support modifying Android captures
+* :construction: : Early Preview / Alpha / Beta
+
+### Cross-Platform Capture/Replay Support
+
+Cross platform support, such as capturing on one operating system and
```

Comment:
I suggest inverting the sense of this sentence, maybe something like "The table above only represents replay on the same platform and driver version on which the file was captured."